### PR TITLE
Make octavia resource plural octavias

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/andrewbays/octavia-operator
```
Comment (dprince) [inline config/manager/kustomization.yaml:15]: how did this get in here!?

Comment (abays) [reply config/manager/kustomization.yaml:15]: :astonished:

**Comment by abays** [comment]
We also need to update to use plurals here:

- https://github.com/openstack-k8s-operators/octavia-operator/blob/main/config/webhook/manifests.yaml (YAML)
- https://github.com/openstack-k8s-operators/octavia-operator/blob/main/api/v1beta1/octavia_webhook.go (annotations)
- https://github.com/openstack-k8s-operators/octavia-operator/blob/main/hack/configure_local_webhook.sh

**Review by abays — APPROVED** [review]
Tested locally and it works

/lgtm

**Comment by abays** [comment]
> @dprince: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/octavia-operator-build-deploy-kuttl 	[fea8633](https://github.com/openstack-k8s-operators/octavia-operator/commit/fea86331a75d4a2f413204cfc80fa978a4a2a5c1) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_octavia-operator/156/pull-ci-openstack-k8s-operators-octavia-operator-main-octavia-operator-build-deploy-kuttl/1684543756449091584) 	true 	`/test octavia-operator-build-deploy-kuttl`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openstack-k8s-operators&repo=octavia-operator&pr=156). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:dprince).

```
error: etcdserver: request timed out
```

/test octavia-operator-build-deploy-kuttl

**Comment by abays** [comment]
> @dprince: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/octavia-operator-build-deploy-kuttl 	[fea8633](https://github.com/openstack-k8s-operators/octavia-operator/commit/fea86331a75d4a2f413204cfc80fa978a4a2a5c1) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openstack-k8s-operators_octavia-operator/156/pull-ci-openstack-k8s-operators-octavia-operator-main-octavia-operator-build-deploy-kuttl/1684556722401710080) 	true 	`/test octavia-operator-build-deploy-kuttl`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openstack-k8s-operators&repo=octavia-operator&pr=156). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:dprince).

```
etcdserver: request timed out
```

:scream_cat: 

/test octavia-operator-build-deploy-kuttl

**Pull request merged**